src/App.tsx

```diff
@@ -0,0 +1,212 @@
+// TODO: See if the below LIB can be removed
```

Comment:
@sagar0211 - things shall work fine without `App.tsx` being present

Reply:
Okay . removed and pushed